starter-code/index.js

```diff
@@ -1,10 +1,44 @@
-var SortedList = function() {};
+class SortedList{
+  constructor(sortedList, length){
+    this.sortedList = [];
+    this.length = this.sortedList.length;
+  }
+
+  add(item){
+    this.sortedList.push(item);
+    this.sortedList.sort((a,b) => {
+    return a - b;
+    })
+    this.length ++;
+  }
+  get(pos){
+  return this.sortedList[pos-1];
+    
+  }
+  max(){
+  return Math.max(...this.sortedList);    
+  }
+  min(){
+  return Math.min(...this.sortedList);    
+  }
+  
+  average(){
+    if(this.sortedList <= 0){
+      return 0;
+    } else {
+      return this.sortedList.sum() / this.length;
```

Comment:
since you have the sum() function within the constructor, all you have to do is put `this.sum()`. Also you would have to divide by length of array, therefore `this.length` should be `this.sortedList.length`